src/app/technical-record-search/test-result.service.spec.ts

```diff
@@ -74,6 +75,18 @@ describe('TestResultService', () => {
     req.flush({ mockObject: 'mock' });
   });
 
+  it('updateTestResults should update', (done) => {
+    const mock = { mockObject: 'mock' };
+    service.updateTestResults('123', {} as VehicleTestResultUpdate).subscribe((res) => {
+      expect(res).toBeDefined();
```

Comment:
Can we clear the mock after each test?
Can we spy on the response to check that it is sending the correct body  '123' or have the correct response?